rankings/models.py

```diff
@@ -9,6 +9,7 @@
 
 class Game(models.Model):
     """Game model, represents a single game."""
+    date_time = models.DateTimeField(auto_now_add=True)
```

Comment:
This requires a migration, and with no default set it will ask you to set a default for previous data when creating the migration.

Can you please create the migration (`python manage.py makemigrations rankings`).

I'd appreciate it if you formatted the field like follows to make diffs easier for future changes (any new kwargs added will show up in the diff, but any old kwargs won't as they won't be seen by Git as being modified unless you have changed it):

```python
date_time = models.DateTimeField(
    auto_now_add=True,
)
```

Also, there will be problems with this migration as I'm sure it'll be called `0003_do_a_thing.py` which will conflict when trying to migrate as the other pull request has one also dependent on `0002`. Just something to be aware of.